docs/designers-developers/developers/richtext.md

```diff
@@ -0,0 +1,119 @@
+# RichText Reference
+
+RichText is a component that allows developers to render a [`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content), providing users with the option to format block content to make it bold, italics, linked, or use other formatting.
+
+The RichText component is extremely powerful because it provides built-in functionality you won't find in other components:
+
+* **Consistent Styling in the Admin and Frontend:** The editable container can be set to any block-level element, such as a `div`, `h2` or `p` tag. This allows the styles you apply in style.css to more easily apply on the frontend and admin, without having to rewrite them in editor.css.
+* **Cohesive Placeholder Text:** Before the user writes their content, it's easy to include placeholder text that's already styled to match the rest of the block editor.
+* **Control Over Formatting Options:** It's possible to dictate exactly which formatting options you want to allow for the RichText field. For example, you can dictate whether to allow the user to make text bold, italics or both.
+
+Unlike other components that exist in the [Component Reference](/packages/components/README.md) section, RichText lives separately because it only makes sense within the block editor, and not within other areas of WordPress.
+
+## Property Reference
+
+For a list of the possible properties to pass your RichText component, [check out the component documentation on Github](/packages/block-editor/src/components/rich-text/README.md).
```

Comment:
Let's use the full URL here, since a relative one won't work when this is on WordPress.org.

```suggestion
For a list of the possible properties to pass your RichText component, [check out the component documentation on Github](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/rich-text/README.md).
```

Reply:
Again, it’s perfectly fine as is.

Reply:
Well, maybe not. It’s very unfortunate. Ideally, this should stay as is and the handbook should point to the branch with the current version of WordPress rather than to master which can have features that might even not be published in the olugin.

Reply:
@nosolosw or @chrisvanpatten - any thought how we should approach it? I'm happy to apply the necessary changes to [Devhub](https://make.wordpress.org/docs/handbook/devhub/) and document it.

Reply:
This link won't work (can be checked with the suggestion above).

In general, links to subdirectories of top-level sections aren't exposed in the handbook. We should expose block-editor components in the handbook, and there is https://github.com/WordPress/gutenberg/issues/15107 to track that. I'm unsure what's the status of that.

For this kind of thing, I see that we're using a link directly to the `master` branch (searched for `master` within the docs folder), so I guess it's OK until we figure how to pin the handbook to a specific version.

Reply:
For this one it sounds like we need to move forward using the absolute URL suggested by @noisysocks. Is that correct?

Reply:
Yes, we should follow what @noisysocks shared.